client/layout/guided-tours/tours/jetpack-plugin-updates-tour/index.tsx

```diff
@@ -30,9 +30,8 @@ export const JetpackPluginUpdatesTour = makeTour(
 		{ ...meta }
 		when={ ( state: AppState ) => {
 			const site = getSelectedSite( state );
-			const isRequestingPlugins = isRequesting( state, site?.ID );
 			const sitePlugin = getPluginOnSite( state, site?.ID, 'jetpack' );
-			const res = ! isRequestingPlugins && !! sitePlugin;
+			const res = !! sitePlugin;
```

Comment:
We've removed the fetching check here; and now how are we certain that we're not fetching the plugins at this time?

Reply:
I tested this and it seemed to work as expected for me. Also, I wasn't sure how to update the code in here to check if we are making the request still.

Reply:
Maybe if we pass props to the `when` callback in addition to the `state`?